fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -738,22 +739,6 @@ public void testCreateContainerWithCharset() throws IOException {
         }
     }
 
-    @Test
-    public void testGetRDFSourceWithPreferMinimal() throws IOException {
```

Comment:
For clarity, I just moved the testing of Preference Applied header into the `testGetObjectGraphMinimal` to reduce the extra work.